x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -41,14 +41,30 @@ type ImmutableMap<K, V> = ReadonlyMap<Immutable<K>, Immutable<V>>;
 type ImmutableSet<T> = ReadonlySet<Immutable<T>>;
 type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 
+export interface EventStats {
+  /**
+   * The total number of related events (all events except process and alerts) that exist for a node.
+   */
+  total: number;
+  /**
+   * A mapping of ECS event.category to the number of related events are marked with that category
+   * For example:
+   *  {
+   *    network: 5,
+   *    file: 2
+   *  }
+   */
+  categories: Record<string, number>;
```

Comment:
:grey_question: naming this property `byCategory` might crystallize its purpose a little better. `categories` makes me thing of a set of category names.